src/frontend/data/stores/useParticipantsStore/index.spec.tsx

```diff
@@ -0,0 +1,56 @@
+import { useParticipantsStore } from '.';
+
+const participant = {
+  isInstructor: true,
+  isOnStage: true,
+  name: 'Instructor',
+};
+
+describe('useParticipantsStore', () => {
+  it('executes useParticipantsStore/addParticipant', () => {
+    expect(useParticipantsStore.getState()).toEqual(
+      expect.objectContaining({
+        participants: [],
+      }),
+    );
+
+    useParticipantsStore.getState().addParticipant(participant);
+
+    expect(useParticipantsStore.getState()).toEqual(
+      expect.objectContaining({
+        participants: [participant],
+      }),
+    );
+  });
```

Comment:
Can you add at least one other participant please.

Reply:
I reworked my tests a little bit, in order to check more use cases and include more participants. commit [bacb30](https://github.com/openfun/marsha/pull/1282/commits/bacb301553a349c790a24b82e96740bc3e0a3f86)